features/steps/steps.go

```diff
@@ -45,6 +45,11 @@ func (c *Component) RegisterSteps(ctx *godog.ScenarioContext) {
 
 	ctx.Step(`^these reindex-requested events are consumed:$`, c.theseReindexrequestedEventsAreConsumed)
 	ctx.Step(`^I should receive a reindex-requested response$`, c.iShouldReceiveAReindexrequestedResponse)
+
+	ctx.Step(`^get published-index request to zebedee is successful$`, c.getPublishedIndexRequestToZebedeeIsSuccessful)
+	ctx.Step(`^get published-index request to zebedee is unsuccessful$`, c.getPublishedIndexRequestToZebedeeIsUnsuccessful)
+	ctx.Step(`^the URLs of zebedee documents are retrieved successfully$`, c.nothingHappens)
```

Comment:
Is this right, should we not return some fake urls?

Reply:
The reason why this is not done is that the app isn't actually returning the fake URLs back to the user. Those URLs are just logged. Also, in the future, these URLs will be sent via a Kafka message to the `content-updated` topic in the near future in which the component test can be updated according to that need